SabaCenter/Model/Models/ScheduleItem.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ScheduleItem.swift
+//  SabaCenter
+//
+//  Created by Mohsen Hosseini on 3/18/18.
+//  Copyright © 2018 Saba. All rights reserved.
+//
+
+import Foundation
+
+class ScheduleItem
+{
+    var date: Date
+    var description: String
+    enum Importance { case normal; case high}
```

Comment:
Where is this enum used? It's only defined, but there's no property of its type.